std-bits/base/src/main/java/org/enso/base/enso_cloud/telemetry/TelemetryLog.java

```diff
@@ -0,0 +1,19 @@
+package org.enso.base.enso_cloud.telemetry;
+
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import org.enso.base.enso_cloud.logging.LogApiAccess;
+
+public final class TelemetryLog {
+  private TelemetryLog() {}
+
+  public static void logAsync(String loggerName, String message, ObjectNode metadata) {
+    var event = TelemetryLogMessage.create(loggerName, message, metadata);
+    LogApiAccess.INSTANCE.logWithoutConfirmation(event);
+  }
+
+  public static class TelemetryLogError extends RuntimeException {
```

Comment:
```suggestion
  public static final class TelemetryLogError extends RuntimeException {
```

Reply:
Done in c0e771c1273cc8ae6e7818805f5c296bdd40d82b